src/rust/scylla-provisioner/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "scylla-provisioner"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+[dependencies]
+async-trait = "0.1.56"
+clap = { version = "3.0", default_features = false, features = ["std", "env"] }
+env_logger = "0.9.0"
+grapl-tracing = { path = "../grapl-tracing" }
+rust-proto = { path = "../rust-proto" }
+scylla = "0.4.7"
+secrecy = "0.8.0"
+thiserror = "1.0.31"
+tokio = { version = "1.20.0", features = ["full"] }
+tracing = "0.1.36"
+uuid = { version = "1.1.2", features = ["v4"] }
+
+[dev-dependencies]
+test-log = { version = "0.2", default-features = false, features = ["trace"] }
```

Comment:
Does this print terminal escape characters?

I've seen them in Rust integration test stdout before, and remember thinking it might have been due to this.

If it is, I suggest we configure it in such a way that doesn't print those, or use something else if that can't be avoided.

Maybe what I saw was the how the tracing subscriber was configured instead? Not sure, but can be addressed later.

Reply:
I'm not sure what's responsible for it tbh